cmd/vela-makisu/global.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+type (
+	// Global represents the global flags that can be set on makisu commands.
+	Global struct {
+		CPU *CPU
+		Log *Log
+	}
+
+	// CPU represents the cpu specific global flags available.
+	CPU struct {
+		// enables viewing the cpu profile of the application
+		Profile bool
+	}
+
+	// Log represents the log specific global flags available.
+	Log struct {
+		// enables setting the format output for the logs - options: (json|console)
+		Fmt string
+		// enables setting the log level for the logs - options: (debug|info|warn|warn)
+		Level string
+		// enables setting the output path for the logs
+		Output string
+	}
+)
+
+var (
+	// configFlags represents the global flags that can be set on the makisu commands.
+	// nolint
+	globalFlags = []cli.Flag{
+		&cli.StringFlag{
+			EnvVars:  []string{"PARAMETER_GLOBAL_FLAGS"},
+			FilePath: string("/vela/parameters/makisu/config/cpu_profile,/vela/secrets/makisu/config/cpu_profile"),
```

Comment:
Are the paths here accurate?

I wonder if they could be named a bit closer to the flag and env var naming 👍 

```suggestion
			FilePath: string("/vela/parameters/makisu/global/flags,/vela/secrets/makisu/global/flags"),
```